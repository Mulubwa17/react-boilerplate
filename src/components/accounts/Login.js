import React from "react";
import { Link } from "react-router-dom";

function Login() {
  return (
    <p className="example-div">
      Welcome to the Login Page{" "}
      <Link to="/register" style={{ textDecoration: "none", color: "#79ad6f" }}>
        {" "}
        Register Here
      </Link>
    </p>
  );
}

export default Login;
