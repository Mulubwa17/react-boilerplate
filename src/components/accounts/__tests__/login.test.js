import React from "react";
import { shallow } from "enzyme";
import Login from "../Login";

describe("login page", () => {
  it("should exists", () => {
    expect(Login).toBeDefined();
  });
  it("login renders without crashing", () => {
    shallow(<Login />);
  });
  it("should render correctly", () => {
    const loginSnap = shallow(<Login />);
    expect(loginSnap).toMatchSnapshot();
  });
  it("should contain a p element", () => {
    const wrapper = shallow(<Login />);
    expect(wrapper.find("p")).toHaveLength(1);
  });
});
