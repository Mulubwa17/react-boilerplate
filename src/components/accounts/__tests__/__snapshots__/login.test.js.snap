// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login page should render correctly 1`] = `
<p
  className="example-div"
>
  Welcome to the Login Page
   
  <Link
    replace={false}
    style={
      Object {
        "color": "#79ad6f",
        "textDecoration": "none",
      }
    }
    to="/register"
  >
     
    Register Here
  </Link>
</p>
`;
